src/content/docs/code-push/guides/patch-signing.mdx

```diff
@@ -7,19 +7,6 @@ sidebar:
 
 {/* cspell:words outform pubout */}
 
-:::note
-
-We've shipped this version of patch signing to unblock certain customer
-evaluations. While we're confident that this feature adds a useful layer of
-security, it does not yet have all the expected integrations with cloud signing,
-keychains, etc. that a final product would expect to have. We also do not yet
-manage signing keys for you or support many key formats or signature types.
```

Comment:
Wondering if the fact that we don't manage your key's for you is worth keeping around so people don't make the assumption that we do like the stores. Thoughts?

Reply:
Updated